### ISSUE TITLE
Getting the familiar black screen bug with OTG on v0.6.1

### ISSUE THREAD

**Issue by ThePwnd** [fraenkelc/amidst-forge#10]
I hate to be the bearer of bad news, especially after reading some of the closed issues where you say you came up with a fix for this issue in v0.6.1..... but I'm running this version of your mod on my client along with Open Terrain Generator v6.  When I launch AMIDST from inside the game, I choose "File > New from Seed," enter the seed from my custom OTG world, and I'm greeted with nothing but a black screen with "Unknown Biomes" everywhere.

Let me know if you need any more info to recreate the problem.

**Comment by fraenkelc** [comment]
What versions of Forge and OTG are you using? 
Is your world gen  / the biome config using the replaceToBiomeName feature?

**Comment by ThePwnd** [comment]
Ok, so I was using Forge 2768, but when you asked me about Forge versions, I thought to try 2838 too because there was another mod I had used recently that needed the latest dev version of Forge to work, but it didn't work either.

So to list them out:

- Forge: 14.23.5.2768 (also tried 14.23.5.2838)
- Open Terrain Generator v6
- Amidst for Forge v0.6.1

Those are the only mods I'm using to test this right now.

I don't believe I'm using the replaceToBiomeName feature.  I generated a vanilla gen preset, but just to be sure, I'll upload it here.
[NewWorld.zip](https://github.com/fraenkelc/amidst-forge/files/3237694/NewWorld.zip)
